e2e/main_test.go

```diff
@@ -0,0 +1,19 @@
+package e2e
+
+import (
+	"flag"
+	"os"
+	"testing"
+)
+
+var (
+	e2ePath = flag.String("e2e-path", ".", "Set path to the e2e directory")
+)
+
+func TestMain(m *testing.M) {
+	flag.Parse()
+	if err := os.Chdir(*e2ePath); err != nil {
```

Comment:
:thinking: why is it needed ? I'm really not a huge fan of doing a `chdir` for executing the tests… :sweat_smile:

Reply:
I believe this will be sorted out in a follow-up?